proxies/venue-graphql-proxy/codegen.yml

```
@@ -0,0 +1,16 @@
+overwrite: true
+schema: http://localhost:${GRAPHQL_PROXY_PORT:-4200}/proxy/graphql
+
+generates:
+  ./src/types/__generated__.ts:
+    plugins:
+      - typescript
+      - typescript-operations
+    config:
+      withHooks: false
+      withComponent: false
+      flattenSelectionSet: true
+    hooks:
+      afterOneFileWrite:
+        - prettier --write
+    ignoreNoDocuments: true,
```

I don't know if this has any effect on anything, but just noticed a trailing comma here:
```suggestion
    ignoreNoDocuments: true
```